06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,193 @@
+/**
```

Comment:
I suspect this is good work, but clean up style violations before I can take a proper look :simple_smile:.